rct229/rulesets/ashrae9012019/section22/section22rule28.py

```diff
@@ -0,0 +1,89 @@
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.ruleset_model_factory import produce_ruleset_model_instance
+from rct229.rulesets.ashrae9012019 import BASELINE_0
+from rct229.rulesets.ashrae9012019.ruleset_functions.get_heat_rejection_loops_connected_to_baseline_systems import (
+    get_heat_rejection_loops_connected_to_baseline_systems,
+)
+from rct229.schema.schema_enums import SchemaEnums
+from rct229.utils.assertions import getattr_
+from rct229.utils.jsonpath_utils import find_all, find_exactly_one_with_field_value
+
+FLUID_LOOP_FLOW_CONTROL = SchemaEnums.schema_enums["FluidLoopFlowControlOptions"]
+
+
+class Section22Rule28(RuleDefinitionListIndexedBase):
+    """Rule 28 of ASHRAE 90.1-2019 Appendix G Section 22 (Chilled water loop)"""
+
+    def __init__(self):
+        super(Section22Rule28, self).__init__(
+            rmrs_used=produce_ruleset_model_instance(
+                USER=False, BASELINE_0=True, PROPOSED=False
+            ),
+            each_rule=Section22Rule28.HeatRejectionRule(),
+            index_rmr=BASELINE_0,
+            id="22-28",
+            description="The baseline building design condenser-water pump shall be modeled as constant volume.",
+            ruleset_section_title="HVAC - Chiller",
+            standard_section="Section 22 CHW&CW Loop",
+            is_primary_rule=True,
+            rmr_context="ruleset_model_descriptions/0",
+            list_path="$.heat_rejections[*]",
+        )
+
+    def create_data(self, context, data):
+        rmd_b = context.BASELINE_0
+        heat_rejection_loop_ids_b = (
+            get_heat_rejection_loops_connected_to_baseline_systems(rmd_b)
+        )
+
+        fluid_loop_b = {
+            heat_rej_b["id"]: find_exactly_one_with_field_value(
+                "$.fluid_loops[*]",
+                "id",
+                getattr_(heat_rej_b, "heat_rejections", "loop"),
+                rmd_b,
+            )
```

Comment:
There is a function called `find_exactly_one_fluid_loop` that does this work.

Reply:
Good point. Addressed.